src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerBinding.cs

```diff
@@ -84,5 +81,31 @@ public ParameterDescriptor ToParameterDescriptor()
                 CollectionName = this._documentCollectionLocation.CollectionName
             };
         }
+
+        internal static bool TryAndConvertToDocumentList(object value, out IReadOnlyList<Document> documents)
+        {
+            try
```

Comment:
nit: 

```c#
documents = null;
try
{
...
// remove else block as documents defaults to null
}
finally
{
  return documents != null;
}
```

Reply:
@mamaso Tried implementing it but I get a "Control cannot leave the body of a finally clause"

Reply:
I removed the else though